package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@al-mabsut/muslimah",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
why are we doing a major semvar? Delete your branch and create a new branch and do:


npm version minor

Reply:
This is minor, the last digit on the right is the patch, the middle is minor and the first one from left is the major

Reply:
Apologies! it should be patch